tests/tools/nnpackage_run/src/rawformatter.cc

```diff
@@ -29,11 +29,13 @@ void RawFormatter::loadInputs(const std::string &filename, std::vector<Allocatio
   uint32_t num_inputs;
   NNPR_ENSURE_STATUS(nnfw_input_size(session_, &num_inputs));
 
-  // TODO: Support multiple inputs
+  // Support multiple inputs
```

Comment:
It should be done in last change about multiple input. If someone would like to make this as another PR, I will do.